libs/sdk-ui-loaders/package.json

```diff
@@ -0,0 +1,107 @@
+{
+    "name": "@gooddata/sdk-ui-loaders",
+    "version": "1.0.0",
+    "description": "GoodData SDK Runtime Component Loaders",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/gooddata/gooddata-ui-sdk.git",
+        "directory": "libs/sdk-ui-loaders"
+    },
+    "author": "GoodData Corporation",
+    "license": "MIT",
+    "main": "dist/index.js",
+    "module": "esm/index.js",
+    "browser": "dist/index.js",
+    "typings": "esm/index.d.ts",
+    "sideEffects": [
+        "*.css",
+        "*.svg"
+    ],
+    "files": [
+        "dist/**/*.js",
+        "dist/**/*.json",
+        "dist/**/*.d.ts",
+        "dist/**/*.map",
+        "dist/**/*.svg",
+        "esm/**/*.js",
+        "esm/**/*.json",
+        "esm/**/*.d.ts",
+        "esm/**/*.map",
+        "esm/**/*.svg",
+        "styles"
+    ],
+    "config": {
+        "eslint": "-c .eslintrc.js --ext ts,tsx src/"
+    },
+    "scripts": {
+        "clean": "rm -rf ci dist esm coverage *.log styles/css && jest --clearCache",
```

Comment:
Please add a `prepack` script since this has custom GoodData licence: https://github.com/gooddata/gooddata-ui-sdk/blob/master/libs/sdk-ui-charts/package.json#L45